weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTable.tsx

```diff
@@ -727,36 +841,64 @@ const ExportRunsTableButton = ({
       alignItems: 'center',
     }}>
     <Button
-      className="mx-16"
+      className="mx-4"
       size="medium"
-      variant="secondary"
-      onClick={() => tableRef.current?.exportDataAsCsv()}
-      icon="export-share-upload">
-      Export to CSV
-    </Button>
+      variant="ghost"
+      onClick={onClick}
+      disabled={disabled}
+      tooltip={
+        exportAll ? 'Export table to CSV' : 'Export selected rows to CSV'
+      }
+      icon="export-share-upload"
+    />
   </Box>
 );
 
 const CompareEvaluationsTableButton: FC<{
   onClick: () => void;
   disabled?: boolean;
-}> = ({onClick, disabled}) => (
+  tooltipText?: string;
+}> = ({onClick, disabled, tooltipText}) => (
   <Box
     sx={{
       height: '100%',
       display: 'flex',
       alignItems: 'center',
     }}>
     <Button
-      className="mx-16"
-      style={{
-        marginLeft: '0px',
-      }}
+      className="mr-4"
       size="medium"
+      variant="ghost"
       disabled={disabled}
       onClick={onClick}
-      icon="chart-scatterplot">
+      icon="chart-scatterplot"
+      tooltip={tooltipText}>
       Compare Evaluations
     </Button>
   </Box>
 );
+
+const DeleteSelectedCallsButton: FC<{
+  onClick: () => void;
+  disabled?: boolean;
+  isRightmostButton?: boolean;
+}> = ({onClick, disabled, isRightmostButton}) => {
+  return (
+    <Box
+      sx={{
+        height: '100%',
+        display: 'flex',
+        alignItems: 'center',
+      }}>
+      <Button
+        className={isRightmostButton ? 'ml-4 mr-16' : 'mx-4'}
```

Comment:
the traces page doesn't have a comparison button, so it gets squished to the edge without this...